test/test_other.py

```diff
@@ -3200,6 +3200,10 @@ def test_embind_tsgen_ignore(self):
     self.emcc(test_file('other/embind_tsgen.cpp'), extra_args)
     self.assertFileContents(test_file('other/embind_tsgen_ignore_3.d.ts'), read_file('embind_tsgen.d.ts'))
 
+    extra_args = ['-sASSERTIONS=0', '-sSTRICT=1', '--no-entry']
+    self.emcc(test_file('other/embind_tsgen.cpp'), extra_args)
+    self.assertFileContents(test_file('other/embind_tsgen_ignore_4.d.ts'), read_file('embind_tsgen.d.ts'))
```

Comment:
Can we not reuse a previous output?  e.g. shouldn't it be the case that enabling/disabling assertions doesn't change the generated bindings?

Reply:
The output is the same with assertions, but the `-sSTRICT=1` makes the output different than `embind_tsgen_ignore_3.d.ts`. It looks like I can actually combine this with `embind_tsgen_ignore_2.d.ts` since that's using minimal runtime and the output will be the same.